packages/lab/src/contact-details/ContactDetails.css

```diff
@@ -56,6 +56,10 @@
   --contact-details-selecting-icon-color: var(--uitk-ratable-background-hover);
 }
 
+.uitk-dark {
+  color: var(--uitk-color-white);
```

Comment:
we shouldn't use colors directly - paul has done contactdetails so can check the appropriate characteristic here https://www.figma.com/file/rkvTiVC2b9fhj7XWJmL4cZ/UITK-Characteristics?node-id=3017%3A220256

Reply:
I believe this should be fixed on a higher level though. Let's have a call next week and discuss. Thanks